logstash-core/lib/logstash/environment.rb

```diff
@@ -40,8 +40,18 @@ module Environment
             Setting::String.new("http.host", "127.0.0.1"),
             Setting::PortRange.new("http.port", 9600..9700),
             Setting::String.new("http.environment", "production"),
+            Setting::String.new("queue.type", "persisted", true, ["persisted", "memory", "synchronous"]),
```

Comment:
synchronous is the current queue. I think we should be able to configure it if is faster and users don't care about reliability  -  and that's an good point, we should think about the one-off use-case, where users just run LS on a single or a set of static files... maybe introduce a cmdline switch for this use-case?